packages/manager/modules/carrier-sip/src/cdr/cdr.constants.js

```diff
@@ -0,0 +1,7 @@
+export const NUMBER_OF_YEARS = 20;
```

Comment:
Number of years for what ?

Reply:
When the user want to get the Call Detail Records of his Carrier SIP service we limit the range of year up to 20.

Reply:
`GET_RECORD_NUMBER_OF_YEARS` then ?

Reply:
It's done. Thanks.